python/tests/test_nessie_cli.py

```diff
@@ -314,16 +315,78 @@ def test_merge() -> None:
 
     # Passing detached commit-id plus a _different_ hash-on-ref --> error
     execute_cli_command(["merge", f"dev@{dev_hash}", "-c", main_hash, "-o", main_hash], ret_val=1)
+    merge_output = execute_cli_command(["merge", "dev", "-c", main_hash])
+
+    branches = ReferenceSchema().loads(execute_cli_command(["--json", "branch"]), many=True)
+    refs = {i.name: i.hash_ for i in branches}
+
+    expected_output_line1 = "The following 1 commits were merged onto main:\n"
+    expected_output_line2 = '{} "test message" \n'.format(refs["dev"])
+    expected_output_line3 = "Resultant hash on main after merge: {}\n".format(refs["main"])
+    assert_that(merge_output).is_equal_to(expected_output_line1 + expected_output_line2 + expected_output_line3)
```

Comment:
sorry i should have been clearer, i suggested to use `merge_output.splitlines()` so that we dont need to use `\n` in these expected messages but can use a list of lines, but you can also leave it as is